src/screens/Account/AccountActions.js

```diff
@@ -39,6 +39,11 @@ export default function AccountActions({ account, parentAccount }: Props) {
   const mainAccount = getMainAccount(account, parentAccount);
   const decorators = perFamilyAccountActions[mainAccount.currency.family];
 
+  const balance =
+    account.type !== "ChildAccount" && account.spendableBalance
+      ? account.spendableBalance
+      : account.balance;
```

Comment:
you can use live-common's helper (from "lib/account") getAccountSpendableBalance or something like this.